### PR TITLE
Siklumhtg-model

### PR DESCRIPTION
**Pull request by bdg-robert**
## Pre-Request Checklist

- [X] Passes rubocop code analysis (try `rubocop --auto-correct`)
- [-] Tests added or adapted (try `rake test`)
- [X] Changes are reflected in the documentation
- [X] User-visible changes appended to [CHANGELOG.md](/CHANGELOG.md)

## Description
Added model file for Siklu Multihaul TG radios which use a different command set than Siklu Etherhaul radios.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - model for Fibrestore (fs.com) FSOS (@tcrichton)
 - model for IP Infusion OcNOS
 - model for Fujitsu 1finity (@jerji)
+- model for Siklu Multihaul TG radios (@bdg-robert)
```
Comment (robertcheramy) [inline CHANGELOG.md:48]: Please enter your change under the [Unreleased] version.

File: docs/Supported-OS-Types.md [hunk 2]
```diff
@@ -146,6 +146,7 @@
 |QTECH               |QSW-2800, QSW-3400, QSW-3450, QSW-3500|[qtech](/lib/oxidized/model/qtech.rb)
 |Quanta              |Quanta / VxWorks 6.6 (1.1.0.8)|[quantaos](/lib/oxidized/model/quantaos.rb)
 |Siklu               |EtherHaul                     |[siklu](/lib/oxidized/model/siklu.rb)
+|                    |Multihaul TG                  |[siklumhtg](/lib/oxidized/model/siklumhtg.rb)
```
Comment (robertcheramy) [inline docs/Supported-OS-Types.md:149]: Please add a link to your model note.

Maybe you could consider becoming a model maintainer?

**Review by robertcheramy — COMMENTED** [review]
Thank you for your PR. I'll be happy to merge it into master when you have taken my review into account.

File: docs/Supported-OS-Types.md (hunk 2 above)
Comment (bdg-robert) [reply docs/Supported-OS-Types.md]: I reviewed the model maintainer role when I was working on this PR. I'm not much of a developer, but I'll try. At a minimum, I can test any changes someone might submit for the models. I listed myself as the maintainer for both models and linked to the note.

**Pull request merged**